roles/installer/ospd/overcloud/hostname/templates/hostnames.yml.j2

```diff
@@ -0,0 +1 @@
+{{ installer.overcloud.hostname.template.content |to_nice_yaml }}
```

Comment:
why do we need a whole file to hold a single line? can't this be done in-line in playbook file?